extensions/librdkafka/ConsumeKafka.h

```diff
@@ -124,6 +124,17 @@ class ConsumeKafka : public core::Processor {
   // Initialize, overwrite by NiFi RetryFlowFile
   void initialize() override;
 
+  class WriteCallback : public OutputStreamCallback {
```

Comment:
Does it need to be public? It's only used inside the class IIRC

Reply:
Indeed it does not need to be, this is just where I found it on the first match when searching similar patterns (ApplyTemplate). Changed it to `private`.